### PR TITLE
probe-run: init at 0.1.8

### PR DESCRIPTION
**Pull request by astro**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/tools/rust/probe-run/default.nix [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{ stdenv, fetchFromGitHub, rustPlatform, pkgconfig, rustfmt, libusb }:
```
Comment (ehmry) [inline pkgs/development/tools/rust/probe-run/default.nix:1]: `libusb1`?

**Comment by astro** [comment]
Replaced `libusb` with `libusb1`, and rebased.

File: pkgs/development/tools/rust/probe-run/default.nix [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  version = "0.1.8";
+
+  src = fetchFromGitHub {
+    owner = "knurling-rs";
+    repo = "probe-run";
+    rev = "v${version}";
+    sha256 = "1jgml8jgnv8a2giblx8jxvfh16y296xm9p6m291i0xcg3fccga70";
+  };
+
+  nativeBuildInputs = [ pkgconfig rustfmt ];
```
Comment (SuperSandro2000) [inline pkgs/development/tools/rust/probe-run/default.nix:14]:
```suggestion
  nativeBuildInputs = [ pkg-config rustfmt ];
```

File: pkgs/development/tools/rust/probe-run/default.nix [hunk 3]
```diff
@@ -0,0 +1,24 @@
+{ stdenv, fetchFromGitHub, rustPlatform, pkgconfig, rustfmt, libusb1 }:
```
Comment (SuperSandro2000) [inline pkgs/development/tools/rust/probe-run/default.nix:1]:
```suggestion
{ stdenv, fetchFromGitHub, rustPlatform, pkg-config, rustfmt, libusb1 }:
```

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review which does not build all packages (e.g. lumo, tensorflow or torch)
If you find some bugs or got suggestions for further things to search or run please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 97481` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
1 package built:
</summary>
<ul>
<li>probe-run</li>
</ul>
</details>

**Comment by evils** [comment]
closing as we now have 0.2.1 due to #119287

**Pull request closed without merging**